pkg/cmd/util/output/backup_describer.go

```diff
@@ -241,3 +247,111 @@ func failedDeletionCount(requests []v1.DeleteBackupRequest) int {
 	}
 	return count
 }
+
+// DescribePodVolumeBackups describes pod volume backups in human-readable format.
+func DescribePodVolumeBackups(d *Describer, backups []v1.PodVolumeBackup, details bool) {
+	if details {
+		d.Printf("Restic Backups:\n")
+	} else {
+		d.Printf("Restic Backups (specify --volume-details for more information):\n")
+	}
+
+	// separate backups by phase (combining <none> and New into a single group)
+	backupsByPhase := groupByPhase(backups)
+
+	// go through phases in a specific order
+	for _, phase := range []string{
+		string(v1.PodVolumeBackupPhaseCompleted),
+		string(v1.PodVolumeBackupPhaseFailed),
+		"In Progress",
+		string(v1.PodVolumeBackupPhaseNew),
+	} {
+		if len(backupsByPhase[phase]) == 0 {
+			continue
+		}
+
+		// if we're not printing details, just report the phase and count
+		if !details {
+			d.Printf("\t%s:\t%d\n", phase, len(backupsByPhase[phase]))
+			continue
+		}
+
+		// group the backups in the current phase by pod (i.e. "ns/name")
+		backupsByPod := new(backupsByPod)
+
+		for _, backup := range backupsByPhase[phase] {
+			backupsByPod.Add(backup.Spec.Pod.Namespace, backup.Spec.Pod.Name, backup.Spec.Volume)
+		}
+
+		d.Printf("\t%s:\n", phase)
+		for _, backupGroup := range backupsByPod.Sorted() {
+			sort.Strings(backupGroup.volumes)
+
+			// print volumes backed up for this pod
+			d.Printf("\t\t%s: %s\n", backupGroup.label, strings.Join(backupGroup.volumes, ", "))
+		}
+	}
+}
+
+func groupByPhase(backups []v1.PodVolumeBackup) map[string][]v1.PodVolumeBackup {
+	backupsByPhase := make(map[string][]v1.PodVolumeBackup)
+
+	phaseToGroup := map[v1.PodVolumeBackupPhase]string{
+		v1.PodVolumeBackupPhaseCompleted:  string(v1.PodVolumeBackupPhaseCompleted),
+		v1.PodVolumeBackupPhaseFailed:     string(v1.PodVolumeBackupPhaseFailed),
+		v1.PodVolumeBackupPhaseInProgress: "In Progress",
+		v1.PodVolumeBackupPhaseNew:        string(v1.PodVolumeBackupPhaseNew),
+		"": string(v1.PodVolumeBackupPhaseNew),
+	}
+
+	for _, backup := range backups {
+		group := phaseToGroup[backup.Status.Phase]
+		backupsByPhase[group] = append(backupsByPhase[group], backup)
+	}
+
+	return backupsByPhase
+}
+
+type podVolumeBackupGroup struct {
+	label   string
+	volumes []string
+}
+
+// backupsByPod stores podVolumeBackupGroups, where the grouping
+// label is "namespace/name".
+type backupsByPod struct {
+	backupsByPodMap   map[string]*podVolumeBackupGroup
+	backupsByPodSlice []*podVolumeBackupGroup
+}
+
+// Add adds a pod volume backup with the specified pod namespace, name
+// and volume to the appropriate group.
+func (b *backupsByPod) Add(namespace, name, volume string) {
+	if b.backupsByPodMap == nil {
```

Comment:
If you wanted to, you could init the map above where you're calling `backupsByPod := new(backupsByPod)`, to avoid this check here. But not necessary.